src/v2/view-builder/components/OdaOktaVerifyTerminalView.js

```diff
@@ -0,0 +1,39 @@
+import { View, loc } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import Enums from 'util/Enums';
+
+export default View.extend({
+  template: hbs`
+    <p class="explanation">
+      {{i18n code="enroll.explanation.p1" bundle="login" arguments="appStoreName"}}
+    </p>
+    <p class="explanation">
+      {{{i18n code="enroll.explanation.p2" bundle="login"}}}
+      <br>
+      {{signInUrl}}
+    </p>
+    <div data-se="app-store">
+      <a href="{{appStoreLink}}" target="_blank">
+        <div class="app-store-logo {{appStoreLogoClass}}"></div>
+      </a>
+    </div>
+  `,
+
+  getTemplateData () {
+    const deviceEnrollment = this.options.appState.get('deviceEnrollment');
+    const platform = (deviceEnrollment.platform || '').toLowerCase();
```

Comment:
could `deviceEnrollment` be null here ? If yes, probably putting a check before accessing it's properties.